mmf/configs/defaults.yaml

```
@@ -41,6 +41,16 @@ training:
     # Tensorboard control, by default tensorboard is disabled
     tensorboard: false
 
+    # Weights and Biases control, by default Weights and Biases (wandb) is disabled
+    wandb: 
+        # Whether to use Weights and Biases Logger, (Default: false)
+        enabled: false
+        # Project name to be used while logging the experiment with wandb
+        wandb_projectname: project
```

```suggestion
        wandb_projectname: mmf_${oc.env:USER}
```

This might be better to be more specific.